# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/moz.build
# Commit: a92c423736e5
# Full Hash: a92c423736e5979a8cf039017fd707d41f665b9c
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-05-25 15:42:48
# Regressor Bug: 1546358
# File Overlap Count: 1
# Description:
#   Bug 1546358 - Forward cache entry check to parent process r=dragana
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D75941
# ==============================================================================

diff -r b21cc935a336 -r a92c423736e5 netwerk/protocol/http/moz.build
--- a/netwerk/protocol/http/moz.build	Fri May 22 13:37:54 2020 +0000
+++ b/netwerk/protocol/http/moz.build	Mon May 25 11:07:07 2020 +0000
@@ -87,6 +87,7 @@
     'BackgroundDataBridgeChild.cpp',
     'BackgroundDataBridgeParent.cpp',
     'CacheControlParser.cpp',
+    'CachePushChecker.cpp',
     'ClassifierDummyChannel.cpp',
     'ClassifierDummyChannelChild.cpp',
     'ClassifierDummyChannelParent.cpp',